app/src/main/java/com/vitorpamplona/amethyst/ui/screen/ChatroomListFeedView.kt

```diff
@@ -77,19 +87,85 @@ private fun FeedLoaded(
 ) {
     val listState = rememberLazyListState()
 
-    LazyColumn(
-        contentPadding = PaddingValues(
-            top = 10.dp,
-            bottom = 10.dp
-        ),
-        state = listState
-    ) {
-        itemsIndexed(state.feed.value, key = { index, item -> if (index == 0) index else item.idHex }) { index, item ->
-            ChatroomCompose(
-                item,
-                accountViewModel = accountViewModel,
-                navController = navController
-            )
+    var markedAllAsRead by remember { mutableStateOf<Boolean>(false) }
+
+    Column {
+        TextButton(
+            modifier = Modifier.fillMaxWidth(),
+            shape = MaterialTheme.shapes.large,
+            onClick = { markedAllAsRead = true }
+        ) {
+            Text(stringResource(R.string.mark_all_as_read))
+        }
```

Comment:
This is mostly a proof of concept, it could use a bit of design work and perhaps an if statement.